fly_manager.py

```diff
@@ -0,0 +1,82 @@
+import json
+import subprocess
+import os
+from set_env import EnvironmentVarSetting
+import sys
+
+
+class FlyHelper:
+    def __init__(self) -> None:
+        pass
+
+    def create_app(self, app_name):
+        print(app_name)
+        # Define your app parameters
+
+        region = "iad"  # default to Virginia
+        database_url = "postgres://user:password@host:port/database"
+
+        # Launch the Fly app and set the DATABASE_URL environment variable
+        # command = f'flyctl launch --name {app_name} --region {region} "'
+        command = f'flyctl launch --name {app_name} --region {region} --env "DATABASE_URL={database_url}"'
+        print(command)
```

Comment:
If this print command is for debugging purpose then please remove it. Such print commands used for debugging should not run in production environment.

Reply:
yes, it's for debugging purpose i will clean up when i finish